apps/cowswap-frontend/src/common/pure/OrderSubmittedContent/index.tsx

```diff
@@ -51,7 +53,11 @@ export function OrderSubmittedContent({ chainId, account, isSafeWallet, hash, on
     <Wrapper>
       <AnimatedConfirmation />
       <Caption>
-        <Trans>Order Submitted</Trans>
+        {safeWallet && safeWallet.threshold > 1 ? (
```

Comment:
Tiny nitpick:

```suggestion
        {safeWallet?.threshold > 1 ? (
```

Same in the other file with a similar expression.

Reply:
Ahhh right, it's comparing to a number, and `undefined` is not one.
Never mind then.